GitUI/CommandsDialogs/FormPull.Designer.cs

```diff
@@ -221,6 +221,8 @@ private void InitializeComponent()
             // 
             this.groupBox2.Anchor = ((System.Windows.Forms.AnchorStyles)(((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Left) 
             | System.Windows.Forms.AnchorStyles.Right)));
+            this.groupBox2.AutoSize = true;
+            this.groupBox2.AutoSizeMode = System.Windows.Forms.AutoSizeMode.GrowOnly;
```

Comment:
Any reason not to set `GrowAndShrink`?

Reply:
Not much. It turns out that the size computed from the designer is the smallest (96DPI and no font scaling) so the size can only grow. I think that GrowAndShrink  works as well

Reply:
I am open to change to GrowAndShrink. Do you see an upside to the change?